### PR TITLE
BLKN: 39 system messages

### LINKED ISSUE #39: BLKN: System messages
There is no directory for system messages in `blkn/src`
https://app.zeplin.io/project/5a8598936eb267528571d4aa

Props:
Type (error | succes | warning | info)
Icon (boolean)
Title (string with the title)

Title could be also specified as separate component with `this.props.children` so it's apperance would depend on implemmenting?

Make sure these boxes are ticked before submitting a new issue:

* [x] I have checked that this is not a duplicate
* [x] I have added a label categorizing this under BLKN | Kiwicom | Icons | Global

### PR DESCRIPTION
**Pull request by prztrz**
This Pull Request meets the following criteria:

* [x] Tests have been added/adjusted for my new feature
* [x] New Components are registered in index.js of my project

Closes #39

### CODE REVIEW

File: blkn/src/SystemMessage/IconWrapper.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// @flow
+import * as React from "react";
+import { colors } from "../constants";
+
+type Props = {
+  type: "warning" | "error" | "success" | "info",
+  icon: Function,
```
Comment (oreqizer) [inline blkn/src/SystemMessage/IconWrapper.js]: type too vague, be specific

File: blkn/src/SystemMessage/IconWrapper.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+const fillColors = {
+  warning: "#f9971e",
+  error: colors.red,
+  success: colors.green,
+  info: colors.azure,
+};
+
+const IconWrapper = (props: Props) => {
+  const { icon: Icon, type } = props;
```
Comment (oreqizer) [inline blkn/src/SystemMessage/IconWrapper.js]: `Icon` used as a class, make the prop PascalCase

File: blkn/src/SystemMessage/Message.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+const fontColors = {
+  warning: "#a93610",
+  error: "#650808",
+  success: "#065d12",
+  info: "#07405c",
+};
+
+const Message = (props: Props) => (
+  <div>
```
Comment (oreqizer) [inline blkn/src/SystemMessage/Message.js]: `<div>` -> `<React.Fragment>`

File: blkn/src/SystemMessage/Title.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+const fontColors = {
+  warning: "#a93610",
+  error: "#650808",
+  success: "#065d12",
+  info: "#07405c",
+};
+
+const Title = (props: Props) => (
+  <div>
```
Comment (oreqizer) [inline blkn/src/SystemMessage/Title.js]: fragment

File: blkn/src/SystemMessage/__tests__/index.test.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+import SystemMessage from "../";
+
+describe("System Message", () => {
+  let component = shallow(<SystemMessage type="success" />);
+
+  it("Should contain a Message ", () => {
+    expect(component.find("Message").exists()).toBe(true);
+  });
+
+  describe("Rendered with icon ", () => {
```
Comment (oreqizer) [inline blkn/src/SystemMessage/__tests__/index.test.js]: imo too much of an overkill to nest this

File: blkn/src/SystemMessage/index.js [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+import * as React from "react";
+import { colors } from "../constants";
+import Message from "./Message";
+import Title from "./Title";
+import IconWrapper from "./IconWrapper";
+
+type Props = {
+  type: "warning" | "error" | "success" | "info",
+  title?: string,
+  icon?: Function,
```
Comment (oreqizer) [inline blkn/src/SystemMessage/index.js]: vague

File: blkn/src/SystemMessage/IconWrapper.js (hunk 1 above)
Comment (prztrz) [reply blkn/src/SystemMessage/IconWrapper.js]: Do you have any suggestions? Label? Outcome? Response?

File: blkn/src/SystemMessage/IconWrapper.js (hunk 2 above)
Comment (prztrz) [reply blkn/src/SystemMessage/IconWrapper.js]: I really do not understand this comment :(, what should be PascalCased?

Comment (prztrz) [reply blkn/src/SystemMessage/IconWrapper.js]: Ok I read your comment from other PR now it's clear thank you

File: blkn/src/SystemMessage/IconWrapper.js (hunk 1 above)
Comment (prztrz) [reply blkn/src/SystemMessage/IconWrapper.js]: Ok shame for me you reffered to icon not type. sorry

File: blkn/src/SystemMessage/IconWrapper.js (hunk 2 above)
Comment (prztrz) [reply blkn/src/SystemMessage/IconWrapper.js]: After long fight with this problem: this just does not work when props is PascalCased. I cannot do antyhing about it.

Comment (oreqizer) [reply blkn/src/SystemMessage/IconWrapper.js]: how doesn't it work? I've never had problems with pascal cased props

File: blkn/src/SystemMessage/index.js [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+  info: colors.blue.light,
+};
+
+const SystemMessage = (props: Props) => (
+  <div>
+    {props.Icon && <IconWrapper type={props.type} Icon={props.Icon} />}
+    {props.title && <Title type={props.type}>{props.title}</Title>}
+    <Message type={props.type}>{props.children}</Message>
+    <style jsx>{`
+                    div {
```
Comment (FilipMessa) [inline blkn/src/SystemMessage/index.js]: fix Indent pls ;)

Comment (FilipMessa) [reply blkn/src/SystemMessage/index.js]: screen shot 2018-03-03 at 11 57 58

**Pull request merged**